original/Makefile

```diff
@@ -13,8 +13,7 @@ TIMER_LIB = ${TIMER_DIR}/dl_timer_lib.a
 INF_DIR = ${SHARED_DIR}/dl_esm_inf/finite_difference
 INF_INC = ${INF_DIR}/src
 INF_LIB = ${INF_DIR}/src/dl_esm_inf_fd.a
-NVCC_INC = /usr/local/cuda-8.0/include/ 
-NVCC_LIB = -L/usr/local/cuda-8.0/lib64/ -lcudart
+NVCC_FLAGS = -O3 -std=c++14 -use_fast_math -arch=$(CUDA_ARCH) --ptxas-options=-v
```

Comment:
Add `-lineinfo` for better CUDA profiling (slightly larger executables, no /negligible performance impact), although this is mutually exclusive with `-G` for device debugging.

Reply:
Thanks for the tip! I've added `-lineinfo` to the flags.